partshop-backend/src/main/java/hu/kosztaauto/partshop/database/Alkatresz.java

```diff
@@ -0,0 +1,34 @@
+package hu.kosztaauto.partshop.database;
+
```

Comment:
delete one new line, we dont need two

Reply:
Will work on multiple empty lines, prettyer is enabled to run on start/save so it's shouldn't occur.